packages/apm/7.16.1/changelog.yml

```diff
@@ -0,0 +1,77 @@
+# newer versions go on top
+#
+# change type can be one of: enhancement, bugfix, breaking-change
+- version: "7.16.1"
+  changes:
+    - description: Added `agent_config_applied` mapping to `metrics-apm.internal` data stream.
+      type: bugfix
+      link: https://github.com/elastic/apm-server/issues/6767
+- version: "7.16.0"
```

Comment:
I was wondering if we should set this to `7.16.1` instead? But given that this is a patch version, I don't see any harm in allowing the patched version for `7.16.0`. Any thoughts?